src/routes/safe/components/Apps/ManageApps.tsx

```diff
@@ -47,15 +57,7 @@ const ManageApps = ({ appList, onAppAdded, onAppToggle }: Props) => {
         <ManageListModal
           addButtonLabel="Add custom app"
           defaultIconUrl={appsIconSvg}
-          formBody={
-            <AddAppFrom
-              formId={FORM_ID}
-              appList={appList}
-              closeModal={closeModal}
-              onAppAdded={onAppAdded}
-              setIsSubmitDisabled={setIsSubmitDisabled}
-            />
-          }
+          formBody={Form}
```

Comment:
I think it will be better if we allow passing this as children (not needed to do now)

Reply:
it could be, but Form, in ManageList component, is optional, so if we pass it as a child, that child will be shown only when the "add" button is clicked.